### DISCUSSION TITLE
Not able to login

### DISCUSSION THREAD

**Discussion by mbaki** [Security-Onion-Solutions/securityonion#3219] [General]
Hi all,

I downloaded security onion 2.3.21 iso image and burned to a USB drive, and the whole day I am trying to figure out why I cannot login using the username and password I assigned when I was asked for an "administrator" username and password during the setup.

When I try to login to continue the setup, it says "login incorrect"

Thanks
Monah

**Comment by weslambert** [comment]
Through what kind of interface are you attempting to install? A desktop, remote management, hypervisor/VM?

**Reply by mbaki** [reply]
Hi Wes,
It’s a dell r240 server. I must have installed it 10 times and always same issue. I am gonna try Monday again and see what happens. Will keep you posted.

**Comment by arnydo** [comment]
I had the same issue. Seems that the username entered into the setup is not what it is wanting.

Try using "root" as the username with the password you configured.

**Comment by leskin** [comment]
I'm having the same problem with the 2.3.40 ISO image. The login error appeared while setting up both the manager and search nodes (so far).  I just tried a simpler password and that seems to be the issue.  I removed the asterisk and exclamation point I had used in the password and the login error message did not occur and I was able to continue with the setup.

**Reply by mbaki** [reply]
My password had a "!" in it, removed it and it worked

**Comment by ZeKay** [comment]
I am having an issue, where i can login to GUI but not able to login to console. I have recently installed 2.3.40. any help?
